doc/source/ray-air/preprocessors.rst

```diff
@@ -0,0 +1,175 @@
+    :end-before: __trainer_end__
+
+.. note::
+
+    If you're passing a Preprocessor that is already fitted, it will be refitted on the ``"train"`` Dataset.
+    Adding the functionality to support passing in a fitted Preprocessor is being tracked
+    `here <https://github.com/ray-project/ray/issues/25299>`__.
+
+Tune
+~~~~
+
+If you're using ``Ray Tune`` for hyperparameter optimization, be aware that each Trial will instantiate its own copy of
+the Preprocessor and the fitting and transformation logic will occur once per Trial.
+
+Checkpoint
+~~~~~~~~~~
+
+``Trainer.fit()`` returns a ``Results`` object which contains a ``Checkpoint``.
+If a Preprocessor was passed into the Trainer, then it will be saved in the Checkpoint along with any fitted state.
+
+As a sanity check, let's confirm the Preprocessor is available in the Checkpoint. In practice you should not need to do this.
+
+.. literalinclude:: doc_code/preprocessors.py
+    :language: python
+    :start-after: __checkpoint_start__
+    :end-before: __checkpoint_end__
+
+
+Predictor
+~~~~~~~~~
+
+A ``Predictor`` can be constructed from a saved ``Checkpoint``. If the Checkpoint contains a Preprocessor,
+then the Preprocessor will be used to call ``transform_batch`` on input batches prior to performing inference.
+
+In the following example, we show the Batch Predictor flow. The same logic applies to the Online Inference flow.
+
+.. literalinclude:: doc_code/preprocessors.py
+    :language: python
+    :start-after: __predictor_start__
+    :end-before: __predictor_end__
+
+Types of Preprocessors
+----------------------
+Ray AIR provides a handful of Preprocessors that you can use out of the box, and more will be added over time.
+`Contributions <https://docs.ray.io/en/master/getting-involved.html>`__ are welcome!
+
+.. tabbed:: Tabular
+
+    #. :class:`Categorizer <ray.ml.preprocessors.Categorizer>`
+    #. :class:`FeatureHasher <ray.ml.preprocessors.FeatureHasher>`
+    #. :class:`LabelEncoder <ray.ml.preprocessors.LabelEncoder>`
+    #. :class:`MaxAbsScaler <ray.ml.preprocessors.MaxAbsScaler>`
+    #. :class:`MinMaxScaler <ray.ml.preprocessors.MinMaxScaler>`
+    #. :class:`Normalizer <ray.ml.preprocessors.Normalizer>`
+    #. :class:`OneHotEncoder <ray.ml.preprocessors.OneHotEncoder>`
+    #. :class:`OrdinalEncoder <ray.ml.preprocessors.OrdinalEncoder>`
+    #. :class:`PowerTransformer <ray.ml.preprocessors.PowerTransformer>`
+    #. :class:`RobustScaler <ray.ml.preprocessors.RobustScaler>`
+    #. :class:`SimpleImputer <ray.ml.preprocessors.SimpleImputer>`
+    #. :class:`StandardScaler <ray.ml.preprocessors.StandardScaler>`
+    #. :class:`SimpleImputer <ray.ml.preprocessors.SimpleImputer>`
+
+.. tabbed:: Text
+
+    #. :class:`CountVectorizer <ray.ml.preprocessors.CountVectorizer>`
+    #. :class:`HashingVectorizer <ray.ml.preprocessors.HashingVectorizer>`
+    #. :class:`Tokenizer <ray.ml.preprocessors.Tokenizer>`
+
+.. tabbed:: Image
+
+    Coming soon!
+
+.. tabbed:: Common APIs
```

Comment:
Could we make this tab the first one? This is to address the feedback that users need to know about these common preprocessors first, before getting into the weeds of built in preprocessors.

Reply:
Yep makes a lot of sense, updated!